lib/assets/javascripts/turbograft/turbolinks.coffee

```diff
@@ -118,15 +180,14 @@ class window.Turbolinks
     triggerEvent 'page:receive'
     options.updatePushState ?= true
 
-    if doc = processResponse(xhr, options.partialReplace)
-      reflectNewUrl url if options.updatePushState
-      nodes = changePage(extractTitleAndBody(doc)..., options)
-      reflectRedirectedUrl(xhr) if options.updatePushState
+    doc = processResponse(xhr, options.partialReplace)
```

Comment:
We're thinking of doing something like
- add something like `data-turbo-asset-name` to our `<script>` tags
- when we find our list of new scripts to add (still just checking if they have a unique src)
    - check if any of the new ones have the same asset name
        - if yes full page refresh
    - if no continue on our merry way